DistantWorlds2.ModLoader.ModManager/ModManager.cs

```diff
@@ -327,22 +332,29 @@ public class ModManager : IModManager {
         // TODO: log
       }
 
-    _loadOrder = Mods.Values.Where(m => m.IsValid)
+    _loadOrder = Mods.Values.Where(m => m.IsValid)      
       .OrderByDescending(m => m.LoadPriority)
+      .ThenByDescending(m => m.Name)
       .StableOrderTopologicallyBy(ModInfo.GetResolvedDependencies);
 
     AppDomain.CurrentDomain.AssemblyResolve += ModAssemblyResolver;
+        
+    Crc64 checksumHasher = new Crc64(); //XxHash returns same results for different input?!?
```

Comment:
If it returns different results for same input, it'd be broken.

If it returns same results for different input, _probably_ the input is broken.

Reply:
https://github.com/dotnet/runtime/issues/69184